setup.py

```diff
@@ -55,13 +47,12 @@ def run(self):
     zip_safe=False,
     url='https://github.com/ua-parser/uap-python',
     include_package_data=True,
-    package_data={'ua_parser': ['regexes.yaml', 'regexes.json']},
+    package_data={'ua_parser': ['regexes.json']},
     setup_requires=['pyyaml'],
-    install_requires=['pyyaml'],
+    install_requires=[],
```

Comment:
plz let me know if you feel strongly against this change since it'd potentially break if someone had `UA_PARSER_YAML` set.

Personally, I'd rather catch the `ImportError`, and message at that point.